pkg/security/rconfig/profiles.go

```diff
@@ -66,18 +66,22 @@ func (r *RCProfileProvider) rcProfilesUpdateCallback(configs map[string]state.Ra
 			continue
 		}
 
-		imageName := utils.GetTagValue("image_name", profile.Tags)
-		imageTag := utils.GetTagValue("image_tag", profile.Tags)
-
+		// get image name from tags of the first version
+		if len(profile.ProfileContexts) == 0 {
```

Comment:
What about having the image name at the top level and just a list of image_tags in the profilecontext tags ?

Reply:
I agree. The change is already present on the `yoanngh/persist-profiles` branch (adding the profile selector on profile top level). I'll ensure to include this change on the second PR.